.github/workflows/elixir.yml

```diff
@@ -127,6 +127,54 @@ jobs:
       - name: Run tests
         run: mix test
 
+  integration:
+    name: Run integration tests
+    runs-on: u22-arm-runner
+    needs: [deps]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Elixir
+        id: beam
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: '25.3.2.7'
+          elixir-version: '1.14.5'
```

Comment:
wondering if we want to put these versions in the env vars

Reply:
Could be useful, but as a separate PR. There I simply copied what we have in other steps.